### PR TITLE
Report mochify errors

### PR DESCRIPTION
**Pull request by islemaster**
After upgrading to Ubuntu 15.10 my copy of phantomjs was broken.  Unfortunately, the initial error that led to this discovery was about as unhelpful as they get:

``` bash
> grunt mochaTest
Running "exec:mochaTest" (exec) task
>> Error: Exit 127
>> Exited with code: 1.
Warning: Task "exec:mochaTest" failed. Use --force to continue.

Aborted due to warnings.
```

Adding `--verbose` didn't help either.  While digging into the issue I discovered that we were catching mochify errors and exiting without printing the error information (which, for some reason, mochify doesn't do internally).  This change was helpful for debugging and won't be intrusive when things are working properly.

### CODE REVIEW

**Comment by islemaster** [comment]
FYI @Bjvanminnen @wjordan

File: apps/test/util/runTests.js [hunk 1]
```diff
@@ -77,7 +77,8 @@ exec(command, function (err, stdout, stderr) {
     transform: 'ejsify',
     'web-security': false
   })
- .on('error', function () {
+ .on('error', function (e) {
+   console.log(e.stack);
```
Comment (Hamms) [inline apps/test/util/runTests.js]: does `console.trace` work here?

Comment (islemaster) [inline apps/test/util/runTests.js]: It does, with slightly more verbose output.  Is that preferred?  Neither seems to be standard.

Here's [`console.trace()`](https://developer.mozilla.org/en-US/docs/Web/API/Console/trace) output:

``` bash
╰○ grunt mochaTest
Running "exec:mochaTest" (exec) task
>> Error: Exit 127
>> Trace
>>     at Browserify.<anonymous> (/home/brad/Projects/code-dot-org/apps/test/util/runTests.js:82:12)
>>     at Browserify.emit (events.js:129:20)
>>     at /home/brad/Projects/code-dot-org/apps/node_modules/mochify/lib/phantom.js:29:11
>>     at /home/brad/Projects/code-dot-org/apps/node_modules/mochify/node_modules/phantomic/lib/phantomic.js:141:9
>>     at null._onTimeout (/home/brad/Projects/code-dot-org/apps/node_modules/mochify/node_modules/phantomic/lib/phantomic.js:59:11)
>>     at Timer.listOnTimeout (timers.js:119:15)
>> Exited with code: 1.
```

And here's [`console.log(error.stack)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Stack):

``` bash
╰○ grunt mochaTest
Running "exec:mochaTest" (exec) task
>> Error: Exit 127
Error: Exit 127
    at /home/brad/Projects/code-dot-org/apps/node_modules/mochify/lib/phantom.js:29:25
    at /home/brad/Projects/code-dot-org/apps/node_modules/mochify/node_modules/phantomic/lib/phantomic.js:141:9
    at null._onTimeout (/home/brad/Projects/code-dot-org/apps/node_modules/mochify/node_modules/phantomic/lib/phantomic.js:59:11)
    at Timer.listOnTimeout (timers.js:119:15)
>> Exited with code: 1.
```

Comment (Hamms) [inline apps/test/util/runTests.js]: trace is nonstandard for the web, but is I believe generally preferred in [nodejs](https://nodejs.org/api/console.html#console_console_trace_message) land.

LGTM with either method, though.

**Comment by Bjvanminnen** [comment]
lgtm. no opinion on log vs. trace. could also do console.error

**Comment by islemaster** [comment]
Sweet, using console.trace().

**Pull request merged**